### PR TITLE
Support USERNAME and PASSWORD in config file

### PR DESCRIPTION
**Pull request by telatin**
Ciao everyone!
I added support to store username and password in the config file, as shown here and in the updated README.

```
[credentials]
GALAXYAPIKEY= 
CLIENTID= 
CLIENTSECRET= 
USERNAME= 
PASSWORD=
```
I can imagine that there might good reasons not to enable this, but in case this could be of interest to someone I opened the PR... No worries if it's discarded :)

### CODE REVIEW

**Comment by telatin** [comment]
Just noticed that this addresses  #15

File: README.md [hunk 1]
```diff
@@ -160,3 +181,26 @@ Note: Downloading files is not recommended if your computer has access to the NG
 
 * Error: Requested resource wasn't found at http://irida.ca/api/...
   > The sample or project that you requested does not exist in the NGS Archive REST API.  Check your options for the project id (-p or --project) and sample id (-s or --sample) and try again.
+
+## Configuration file
+
+A configuration file, structured as shown below,
+can be used to feed URLs (`BASEURL` and `GALAXYURL`) and the Galaxy API key (`GALAXYAPIKEY`).
```
Comment (ericenns) [inline README.md]: I am curious why GALAXY and GALAXYAPIKEY were included in this example as I cannot find any references to it within the code.

**Review by ericenns — CHANGES_REQUESTED** [review]
Hi @telatin, sorry for the long delay in responding to this. I just have some minor comments below.

File: README.md (hunk 1 above)
Comment (telatin) [reply README.md]: Will check, most likely it's something which can be zapped as only matters at QIB :) brb

**Review by ericenns — APPROVED** [review]
👍 Thanks for your contributions

**Pull request merged**